### PR TITLE
Rahu ifix 230

### PR DESCRIPTION
**Pull request by rahu-eGov**
Fiscal Event parameter validation.

### CODE REVIEW

File: domain-services/fiscal-event-service/src/main/java/org/egov/util/MasterDataConstants.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package org.egov.util;
+
+public class MasterDataConstants {
+    public static final String PROJECT_ID = "PROJECT_ID";
+    public static final String JSONPATH_ERROR = "JSONPATH_ERROR";
+    public static final String FISCAL_EVENT = "FISCAL_EVENT";
+    public static final String REFERENCE_ID = "REFERENCE_ID";
+    public static final String EVENT_TYPE = "EVENT_TYPE";
+
+    private MasterDataConstants() {
```
Comment (rushang7-eGov) [inline domain-services/fiscal-event-service/src/main/java/org/egov/util/MasterDataConstants.java:10]: Remove unused constructor

File: domain-services/fiscal-event-service/src/main/java/org/egov/util/MasterDataConstants.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    public static final String GOVERNMENT_ID = "GOVERNMENT_ID";
+    public static final String TENANT_ID = "TENANT_ID";
+    public static final String EAT_ID = "EAT_ID";
+    public static final String DEPARTMENT_ID = "DEPARTMENT_ID";
+    public static final String LOCATION_ID = "LOCATION_ID";
+    public static final String REQUEST_HEADER = "requestHeader";
+
+    public static final String CRITERIA_TENANT_ID = "tenantId";
+
+    public static final String $_TENANT_LIST = "$.government.*";
```
Comment (rushang7-eGov) [inline domain-services/fiscal-event-service/src/main/java/org/egov/util/MasterDataConstants.java:25]: Any reason to put $ sign?

**Review by rushang7-eGov — COMMENTED** [review]
@pintu-eGov @rahu-eGov

File: domain-services/fiscal-event-service/src/main/java/org/egov/util/MasterDataConstants.java (hunk 1 above)
Comment (rahu-eGov) [reply domain-services/fiscal-event-service/src/main/java/org/egov/util/MasterDataConstants.java:10]: It is not unused, it is created to prohibit new instance creation while this class is only meant for public static final variable.

File: domain-services/fiscal-event-service/src/main/java/org/egov/util/MasterDataConstants.java (hunk 2 above)
Comment (rahu-eGov) [reply domain-services/fiscal-event-service/src/main/java/org/egov/util/MasterDataConstants.java:25]: $ removed

File: domain-services/fiscal-event-service/src/main/resources/application.properties [hunk 3]
```diff
@@ -6,12 +6,12 @@ app.timezone=UTC
...
 
 # KAFKA SERVER CONFIGURATIONS
-#kafka.config.bootstrap_server_config=localhost:9092
+kafka.config.bootstrap_server_config=localhost:9092
 #spring.kafka.consumer.value-deserializer=org.egov.boundary.consumer.HashMapDeserializer
 #spring.kafka.consumer.key-deserializer=org.apache.kafka.common.serialization.StringDeserializer
-#spring.kafka.consumer.group-id=egov-location
-#spring.kafka.producer.key-serializer=org.apache.kafka.common.serialization.StringSerializer
-#spring.kafka.producer.value-serializer=org.springframework.kafka.support.serializer.JsonSerializer
+spring.kafka.consumer.group-id=egov-location
```
[the remark this answers is not in the document]
Comment (rahu-eGov) [reply domain-services/fiscal-event-service/src/main/resources/application.properties:12]: Done

File: domain-services/fiscal-event-service/src/main/resources/application.properties [hunk 4]
```diff
@@ -26,4 +26,18 @@ org.egov.detailed.tracing.enabled=true
 #kafka.producer.config.buffer_memory_config=33554432
 
 ## Kafka Topic Configurations ##
-fiscal.kafka.push.topic=push-fiscal-request
+fiscal.kafka.push.topic=fiscal-event-request
```
[the remark this answers is not in the document]
Comment (rahu-eGov) [reply domain-services/fiscal-event-service/src/main/resources/application.properties:29]: Done

**Pull request merged**